coord/src/dist_aggregate.c

```diff
@@ -200,9 +201,9 @@ typedef struct {
 
 static int getNextReply(RPNet *nc) {
   if (nc->cmd.forCursor) {
-    // if there is only 1 reply left (or none), trigger READs at the shards.
-    // TODO: This hardcoded 1 could be replaced with a configurable value (global or query specific)
-    if (!MR_ManuallyTriggerNextIfNeeded(nc->it, 1)) {
+    // if there are no more than `clusterConfig.cursorReplyThreshold` replies, trigger READs at the shards.
+    // TODO: could be replaced with a query specific configuration
```

Comment:
So not need for this TODO now right?

Reply:
I changed it from a general config to a query-specific option (e.g. `FT.AGGREGATE ... TIMEOUT 1000 REPLY_THRESHOLD 3`)